elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -53,16 +53,16 @@ def render(element_html, data):
     allow_complex = pl.get_boolean_attrib(element, 'allow-complex', ALLOW_COMPLEX_DEFAULT)
     imaginary_unit = pl.get_string_attrib(element, 'imaginary-unit-for-display', IMAGINARY_UNIT_FOR_DISPLAY_DEFAULT)
 
+    operators = ['cos', 'sin', 'tan', 'exp', 'log', 'sqrt', '( )', '+', '-', '*', '/', '^', '**']
+    constants = ['pi', 'e']
+
     if data['panel'] == 'question':
         editable = data['editable']
         raw_submitted_answer = data['raw_submitted_answers'].get(name, None)
 
-        operators = ', '.join(['cos', 'sin', 'tan', 'exp', 'log', 'sqrt', '( )', '+', '-', '*', '/', '^', '**'])
-        constants = ', '.join(['pi, e'])
-
         info_params = {
             'format': True,
-            'variables': variables_string,
+            'variables': variables,
```

Comment:
Was `variables_string`  never used before? I'm not seeing a corresponding change above.

Reply:
`variables_string` is used to populate `variables`, the way this is rendered has been changed though